Makefile

```diff
@@ -36,8 +36,9 @@ install-deps:
 	sudo mv wasi-sdk-15.0/* /opt/wasi-sdk/
 	sudo rm -rf wasi-sdk-*
 	sudo apt-get update
-	sudo apt-get install lsb-release
-	sudo apt-get install -y redis-server
+	sudo apt-get install lsb-release -y
+	sudo apt-get install redis-server -y
+	sudo apt-get install protobuf-compiler -y
```

Comment:
Why are we adding this dependency?

Reply:
It's because the newest Redis depend on protob